SpywareFilter/sections/specific.txt

```diff
@@ -23,6 +23,9 @@
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 ||googleadservices.com/pagead/conversion_async.js$redirect=noopjs,domain=ffm.to,important
 !
+||static.danawa.com/*/adweb_counter_danawa.min.js
+||*.danawa.com/*/*logger*.js
```

Comment:
Better block instead of all `logger`s. I see no requests from that scripts.
http://static.danawa.com/globaljs/static/logger_Insight_WebAnalytics.js
http://static.danawa.com/globaljs/static/logger_trackEvent.js

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/76128/commits/4bd74e3760dfa08dcbc6f5d3f95d09a33bdd8d3e